events-api.go

```diff
@@ -75,7 +83,7 @@ func (e *EventsApi) flush() {
 		for _, event := range openapiEvents {
 			jsonEvent, err := json.Marshal(event)
 			if err != nil {
-				log.Printf("[KABLE] Kable Event (Error): %s", event)
+				log.Printf("[KABLE] Kable Event (Error): %T", event)
```

Comment:
What is `T`?

Reply:
Great question. I misunderstood the documentation the correct formatter we want in this case is `%+v`. You can see this change in the commit: f395cc8. This will render like the following:
```
type Tree struct{
    name string
}
...
// includes names inside struct
    fmt.Printf("%+v\n", cherry)   // {name:cherry}
```

Reply:
It's also worth noting that this will only ever log when we fail to construct json from our events.